src/components/Modal/VotesList.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+import Grid from "semantic-ui-react/dist/commonjs/collections/Grid/Grid"
+import Avatar from 'decentraland-gatsby/dist/components/User/Avatar'
+import { Address } from 'decentraland-ui/dist/components/Address/Address'
+import { Vote } from '../../entities/Votes/types'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+
+import './ProposalModal.css'
+import './VotesList.css'
+
+const SI_SYMBOL = ["", "k", "M", "G", "T", "P", "E"]
+
+function abbreviateNumber(number: number) {
+
+  const tier = Math.log10(Math.abs(number)) / 3 | 0
+  
+  if (tier == 0) return number
+
+  const suffix = SI_SYMBOL[tier];
+  const scale = Math.pow(10, tier * 3)
+
+  const scaled = number / scale
+
+  return scaled.toFixed(1) + suffix
+}
+
+export type VotesListModalProps = Omit<ModalProps, 'children'> & {
+  onClickAccept?: (e: React.MouseEvent<any>) => void
+  votes?: Record<string, Vote> | null,
+}
+
+export function VotesList({onClickAccept, votes, ...props }: VotesListModalProps) {
+  const l = useFormatMessage()
+
+  return <Modal {...props} size="tiny" className={TokenList.join(['ProposalModal', 'VotesList' ,props.className])} closeIcon={<Close />}>
+  <Modal.Content className="ProposalModal__Title">
+    <Header>{Object.entries(votes || {}).length} {Object.entries(votes || {}).length === 1 ? 'Vote' : 'Votes'}</Header>
```

Comment:
move this to the `en.json#modal.votes_lis.title` and use `l('modal.votes_lis.title', { votes })`